### DISCUSSION TITLE
How can I connect to the Shopware 6 API from another docker container

### DISCUSSION THREAD

**Discussion by Walter-Melon** [shyim/shopware-docker#182] [Q&A] [answered]
Hey I'm currently stuck trying to connect my other docker container to a dev shop so that I can connect to the API.

I have a Laravel application running on http://localhost:8080 (created using https://laravel.com/docs/9.x/sail) 
and my shop on http://sw6.dev.localhost.

Is there some kind of way for me to connect both containers so that I can call the  Admin API?

**Comment by shyim** [comment, marked as answer]
You could join one of the containers in the same network. So they can reach using the internal ip. 
https://docs.docker.com/engine/reference/commandline/network_connect/
